hapi-loader/application.docker.yaml

```diff
@@ -1,6 +1,4 @@
 spring:
-  main:
-    web-application-type: none
```

Comment:
Does this have any implications?

Reply:
not much, a dependency to spring-boot-web has been added, and the hapi-loader service now exposes a /actuator route. It seems that there are no other consequences (calling other routes lead to 404 responses)